src/app/core/status/status.component.ts

```diff
@@ -32,7 +32,10 @@ export class StatusComponent implements OnInit {
       .subscribe(connections => this.peerListCount = connections)
 
     this._rpc.state.observe('encryptionstatus')
-      .subscribe(status => this.encryptionStatus = status);
+      .subscribe(status => {
```

Comment:
This is in the wrong place... It should be in rpc-state.service, that is where we do state modifcations, lets keep it in the same place :)